code/modules/power/apc.dm

```diff
@@ -230,24 +237,24 @@
 	..()
 	if (in_range(user, src))
 		if(stat & BROKEN)
-			to_chat(user, "Looks broken.")
+			to_chat(user, "Похоже, не работает.")
 			return
 		if(opened != APC_COVER_CLOSED)
 			if(has_electronics && terminal)
-				to_chat(user, "The cover is [opened == APC_COVER_REMOVED ? "removed" : "open"] and the power cell is [ cell ? "installed" : "missing"].")
+				to_chat(user, "Крышка [opened == APC_COVER_REMOVED ? "снята" : "открыта"] и [ cell ? "установлен" : "извлечён"] аккумулятор.")
 			else if(!has_electronics && terminal)
-				to_chat(user, "There are some wires but no any electronics.")
+				to_chat(user, "Проводка подключена, но не хватает платы.")
 			else if(has_electronics && !terminal)
-				to_chat(user, "Electronics installed but not wired.")
+				to_chat(user, "Плата установлена, но проводка не подключена.")
 			else
-				to_chat(user, "There is no electronics nor connected wires.")
+				to_chat(user, "Не хватает платы и проводки.")
 		else
 			if(stat & MAINT)
-				to_chat(user, "The cover is closed. Something wrong with it: it doesn't work.")
+				to_chat(user, "Крышка закрыта, но, похоже, щиток сломан.")
 			else if(malfhack)
-				to_chat(user, "The cover is broken. It may be hard to force it open.")
+				to_chat(user, "Крышка помята и не поддаётся.")
```

Comment:
Тут вот кстати тоже нюанс - экзамайн это прежде всего визуальный осмотр, как правило он не должен содержать прямое физическое действие. В английском варианте это учтено но выглядит как мысли в слух, в русском же "и не поддается" очевидно мы пробуем подергать крышку.
Кстати говоря, если теперь русский вариант перевести на английский то первая половина будет как старая но вторая вместо "It may be hard to force it open" будет как "It won't budge".

Вообще broken это не факт что имеется ввиду физическое повреждение, может просто техника выключена и не работает или работает но на экране скажем ошибка. И я почему-то склоняюсь больше что тут упоминание о том что крышку будет сложно открыть, это потому что возможно крышка открывается в режиме работы по кнопке, а теперь т.к мы не имеет доступ к функции открытия, придется применять силу чтобы вскрыть.

Это так мысли в слух.

Reply:
Крышка открывается ломом

Reply:
а это реально examine, не ? Тогда неправ, redo. Зачем там inrange(объясните для глупого)

Reply:
Этот отрезок лежит в длинной тепочке метода "/obj/machinery/power/apc/examine(mob/user)"

Reply:
взломай емагом
и осмотри

Reply:
> как ИИ помял крышку..
перегрел и крышка оплавилась :dovolen:

Ну а как иначе он её блять сломал так что ломом не открыть? Я хз вообще.

Reply:
проблема оригинала, они существуют, или замок из админского сплава с ЦК

Reply:
Да пофиг, малфа всё равно не существует 😈